docs/howto/openstack/nova/move-server-between-regions.md

```diff
@@ -0,0 +1,364 @@
+# Moving a server from one region to another
+
+This guide will show you how to move a server to a different region in {{brand}}.
+
+## Prerequisites
+
+In order to move a server from one region to another, you will need
+
+- a correctly installed and configured [OpenStack CLI client](../../getting-started/enable-openstack-cli.md),
+- access to the RC files with credentials for both the source and target region,
+- enough space on the local machine to be able to download an image of your server,
+- a sufficiently configured target region including a [virtual network](../neutron/new-network.md) and necessary [security groups](../neutron/create-security-groups.md).
```

Comment:
I have pushed a suggested reformat/rewrite for this bullet list in [d9790d6](https://github.com/citynetwork/docs/pull/73/commits/d9790d62cefa9fe9b017e70386d100a9495b9fd6), which we can subsequently squash into the top commit in this PR.